holistics/holistics.py

```
@@ -27,58 +27,51 @@ def __init__(self, api_key, path = None, url = None):
-            res.raise_for_status()
-        except requests.exceptions.HTTPError as err:
-            print (err)
-            return 0
+    def get_url(self, tail_url, params=None):
+        res = requests.get(self.url + tail_url, params = params, headers = self.headers)
+        res.raise_for_status()
         return res
-        
 
-    def SubmitReport (self, report_id, filters=None):
+    def submit_report (self, report_id, filters=None):
         print ("Submitting export request... ", end='')
         tail_url = '/queries/'+str(report_id)+'/submit_export.csv'
-        res = self.GetURL(tail_url, filters)
-        if res!=0:
+        try:
+            res = self.get_url(tail_url, filters)
             res = res.json()
             print ("Success")
-            self.page['job_id'] = str(res['job_id'])
-            return 1
-        else:
-            return 0
+            return str(res['job_id'])
+        except requests.exceptions.HTTPError as err:
+            return err
 
-    def GetExportResults(self, job_id, _page_size = None, _page = None):
+    def get_export_result(self, job_id, _page_size = None, _page = None):
         print ("Getting export results... ", end='')
         tail_url = '/queries/get_export_results.json'
         self.page['job_id']=job_id
         if _page_size is not None:
             self.page['_page_size']=_page_size
         if _page is not None:
             self.page['_page']=_page
-        res = self.GetURL(tail_url, self.page)
-        if res!=0:
+        try:
+            res = self.get_url(tail_url, self.page)
             res = res.json()
             while (res['status'] != 'success'):
                 if res['status'] == 'already_existed':
                     self.page['job_id'] = str(res['job_id'])
-                    break
                 if res['status'] == 'failure':
                     print ("Status: Failure")
                     return 0
```

instead of returning `0`, can raise an Exception here.